sdk/eventgrid/Azure.Messaging.EventGrid/src/Customization/EventGridClientOptions.cs

```diff
@@ -44,9 +49,11 @@ public enum ServiceVersion
         /// The <see cref="ServiceVersion"/> of the service API used when
         /// making requests.
         /// </param>
-        public EventGridClientOptions(ServiceVersion version = LatestVersion)
+        /// <param name="objectSerializer"> Used to serialize the given events to JSON. </param>
+        public EventGridClientOptions(ServiceVersion version = LatestVersion, ObjectSerializer objectSerializer = default)
```

Comment:
Ah, I missed that you were including this in the ctor. I think we generally leave optional properties out of the client options ctor with the exception of the service version. This way, if we have new options in the future, we don't have to worry about adding new overloads.